### PR TITLE
[ID] Add `Play style/Hover`

### PR DESCRIPTION
**Pull request by Fhoxydeo**

### CODE REVIEW

File: wiki/Play_style/Hover/id.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - hovering
+  - play style
+  - gaya bermain
+---
+
+# Hover
+
+**Hovering** adalah istilah yang digunakan untuk mengambarkan tindakan menavigasi (atau "membidik") sebuah pena [tablet](/wiki/Gameplay/Input_device/Graphics_tablet) melewati tablet dengan memegang pena di atas permukaan tablet ketika masih berada di dalam hover zone. Dengan demikian Anda "mengangkat" pena di atas tablet. Ini adalah metode navigasi pena yang berlawanan dengan [dragging](/wiki/Play_style/Drag). Itu dapat ditemukan oleh seluruh pemain dari semua tingkat keahlian, meskipun paling sering dikaitkan dengan pemain yang sangat terampil.
+
+Argumen yang mendukung hovering sering menganggap bahwa itu dapat mengurangi kebutuhan untuk membeli nib pengganti, mengurangi kebutuhan akan tablet cover, dan (secara teknis) "lebih cepat" karena kurangnya gesekan. Sedangkan argumen yang menentang hovering sering mengklaim bahwa jauh lebih sulit untuk mengarahkan secara akurat pada [jump](/wiki/Beatmap/Pattern/Jump) dan sudut yang benar.
```
Comment (Nivalyx) [inline wiki/Play_style/Hover/id.md:14]: perhaps try sth like

`Gaya bermain ini banyak diminati karena cenderung tidak merusak *tablet cover* dan *nib* pada pena serta (secara teknis) "lebih cepat" karena tidak melibatkan banyak gesekan. Di sisi lain, gaya bermain ini diklaim jauh lebih sulit untuk mengarahkan...`

**Comment by Nivalyx** [comment]
would also like to add that directly translating words such as "This", "These", "Thus", etc. in the beginning of the sentence as "Ini" or "Itu" isn't really recommended tho because it'll only make the translated sentence to be... grammatically off if any.

alternatives such as "Hal ini...", "Tindakan ini...", and so on are much much much more proper in like 99% of the case 🐇

image

**Review by Nivalyx — APPROVED** [review]
should be alright!

**Review by TicClick — APPROVED** [review]
Niva knows what's up

**Pull request merged**